Client/DataAccess/SQL/HistorySqlite.swift

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
We keep having a hard time keeping these comments indented properly. You guys should use my Mozilla Xcode Swift template!